pkgs/tools/package-management/rnix-hashes/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, rustPlatform, fetchFromGitHub, fetchpatch }:
+rustPlatform.buildRustPackage rec {
+  pname = "rnix-hashes";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "numtide";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "LYxBNsI3A41I8XeopX2s5UJxwrOkUkRLWTGWn8U+zH4=";
+  };
+
+  cargoSha256 = "Z4ejlg9+q6OzvzIkfstmRpI5mAj8MZzm/op2FQdkl3E=";
+
+  meta = with lib; {
+    description = "Nix Hash Converter";
+    homepage = "https://github.com/numtide/rnix-hashes";
+    license = licenses.asl20;
+    maintainers = with maintainers; [ rizary ];
```

Comment:
You can't add yourself as maintainer here if you're not listed in `maintainers/maintainer-list.nix`.